Formula/automysqlbackup.rb

```diff
@@ -56,4 +58,7 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
```

Comment:
Can this test be modified to do something e.g. run `--version` or `--help` at a minimum? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!